android/BOINC/app/src/main/java/edu/berkeley/boinc/client/ClientStatus.java

```diff
@@ -490,7 +490,7 @@ public String getCurrentStatusDescription() {
 					statusString = ctx.getString(R.string.suspend_battery_overheating);
 					break;
 				case BOINCDefs.SUSPEND_REASON_USER_ACTIVE:
-					Boolean suspendDueToScreenOn = false;
+					Boolean suspendDueToScreenOn;
```

Comment:
Could this and next line be combined into one?

Reply:
Done.